### DISCUSSION TITLE
User Choice

### DISCUSSION THREAD

**Discussion by W0rthlessS0ul** [W0rthlessS0ul/nRF24_jammer#12] [Ideas]
Hello!

The name of my repository states that it is a **jammer**, but I would like to make the project more versatile. I'm considering adding features like:

- **BLE spamer**
- **Mousejacker**
- **Deauther**
- **Other functionalities**

These additions would **change the concept** of my repository.

## **Your Feedback**

What do you think? 

- If you believe this would be **detrimental** to the project, please give a thumbs down: 👎
- If you think it’s a **good idea**, please give a thumbs up: 👍

I will **definitely take your opinions into account** and decide whether to add these features or not. The final decision is **entirely up to you!**

I would also appreciate your **comments** and **thoughts**!

**Comment by Cloudplayer** [comment]
Good idea !

**Comment by samukas81** [comment]
Very goooood idea

**Comment by S7450N** [comment]
hi! i made your jammer with CDEBYTE E01-2G4M27D module. works great! thanks to you! also i made RTL8720dn-Deauther with external antenna ( https://github.com/tesa-klebeband/RTL8720dn-Deauther ). 
i plan to combine these 2 devices in one case. here is an idea for you. Discard esp32 and implement a universal device on RTL8720dn: 
RTL8720dn + external antenna –> dualband wifi deauther (2.4 gHz + 5 gHz)
RTL8720dn + CDEBYTE E01-2G4M27D + external antenna –> jammer (ble, Bluetooth, wifi 2.4 gHz, drones, zigbee etc)
